hybrid/decomposers.py

```diff
@@ -1029,7 +1029,7 @@ def _make_kings_lattice(dimensions: Tuple[int, int],
 
 def make_origin_embeddings(qpu_sampler=None, lattice_type=None,
                            problem_dims=None, reject_small_problems=True,
-                           allow_unyielded_edges=False):
+                           allow_unyielded_edges=False, exclude_dims=set()):
```

Comment:
Avoid [mutable default values](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments). Instead, default to a sentinel value, like `None`, then set to `set()` if it's `None`.